index.html

```diff
@@ -201,38 +209,20 @@
-                        <p>Participants are actively engaged in shaping the program through a strategic, multi-sector partnership led by the <a href="http://www.idrc.ocad.ca" title="Inclusive Design Research Centre" target="_blank">Inclusive Design Research Centre</a> at <a href="http://www.ocadu.ca" title="OCAD University" target="_blank">OCAD University</a>.</p>
                     </div>
 
-                    <div class="small-12 medium-6 column">
-
-                        <p>Outside-IN partners include:</p>
-
-                        <ul>
-                            <li> <a href="http://www.yes.on.ca/" title="Youth Employment Services" target="_blank"> Youth Employment Services </a> </li>
-                            <li> <a href="https://www.tigweb.org/" title="TakingIT Global" target="_blank"> TakingITGlobal </a> </li>
-                            <li> <a href="http://socialinnovation.ca/" title="Centre for Social Innovation" target="_blank"> Centre for Social Innovation </a> </li>
-                            <li> <a href="http://www.riseassetdevelopment.com/" title="RISE Asset Development" target="_blank"> RISE Asset Development </a> </li>
-                            <li> <a href="http://inclusivedesign.ca/" title="Inclusive Design Institute" target="_blank"> Inclusive Design Institute</a> researchers from Seneca, George Brown and Sheridan Colleges</li>
-                        </ul>
-
-                        <p>The project is further supported by partnerships with a large network of obligated organizations that require the skills and competencies being developed through the project, including:</p>
-
-                        <ul>
-                            <li> <a href="http://www.ontario.ca/ministry-government-services" title="Ministry of Government Services" target="_blank"> Ministry of Government Services </a> </li>
-                            <li> <a href="https://www.accessola.org/" title="Ontario Libray Association"> Ontario Library Association </a> </li>
-                            <li> <a href="http://www.bell.ca/" title="Bell" target="_blank"> Bell</a> </li>
-                        </ul>
-
-                    </div>
+                   <div class="small-12 medium-6 column">
+                        <p>Participants are actively engaged in shaping the program through a strategic, multi-sector partnership led by the Inclusive Design Research Centre at OCAD University.</p>
+                   </div>
 
                 </section> <!-- end outside-in-partnersRow -->
 
                 <section class="row outside-in-aboutPartnersRow outside-in-rowPadding">
+                
                     <div class="small-12 column">
-                        <h3>About Our Partners</h3>
+                        <h3>About our partners</h3>
```

Comment:
All of the links in the "About our partners" section are missing hover styling in the high-contrast themes.